makefile

```diff
@@ -99,9 +99,10 @@ htmlCoverageOutput := $(foreach target,$(testPackages),$(buildDir)/output.$(targ
 protocVersion := 3.19.3
 protoOS := $(shell uname -s | tr A-Z a-z)
 ifeq ($(protoOS),darwin)
-protoOS := osx
-endif
+protoOS := osx-x86_64
```

Comment:
This is for M1 users.

Reply:
It doesn't exist for this version of protoc so we have to use the `x86_64` flavor.

Reply:
ah I get it.
Why are we stuck on such an old version?

Reply:
We tried updating and it broke everything, I think there is a ticket tracking it. But I think the original reason was resmoke couldn't upgrade gRPC if I remember correctly (related to updating Python)—this is no longer an issue obviously but I think we shouldn't upgrade until we figure out what we are going to do with Cedar.